### ISSUE TITLE
Provide JSON-LD format of ontology

### ISSUE THREAD

**Issue by alliyya** [cwrc/ontology#471]

**Comment by HannahJS99** [comment]
@alliyya Is this something you're still planning to do, or can it be closed?

**Comment by alliyya** [comment]
Yes, still planning on doing this. The library that provides the serialization now supports exports to JSON-LD so will be an easy addition.
